modules/ivc_champva/spec/jobs/missing_form_status_job_spec.rb

```diff
@@ -32,4 +33,18 @@
 
     IvcChampva::MissingFormStatusJob.new.perform
   end
+
+  context 'Feature champva_failure_email_job_enabled=true' do
+    before do
+      allow(Flipper.champva_failure_email_job_enabled).to receive(:enabled?).and_return(true)
```

Comment:
I was seeing an error `Please stub a default value first if message might be received with other args as well`, so I updated the `with` to pass a `nil` alongside the feature toggle name and that seemed to get everything running fine. Thanks!

Reply:
oh, okay. That might be because when you enable it in `missing_form_status_job.rb`, you pass in `@current_user`:
```ruby
if Flipper.enabled?(:champva_failure_email_job_enabled, @current_user)
```

Reply:
@rmtolmach I've updated it to use `@current_user` in the test - definitely clearer than just using `nil`. Ready for the (hopefully) final approval! Thanks